pages/common/nc.md

```diff
@@ -30,10 +30,10 @@
 
 `nc -q {{timeout}} {{ip_address}}`
 
-- Port scanning:
+- Scan open ports on host:
 
 `nc -v -z {{ip_address}} {{port}}`
 
-- Proxy and port forwarding:
+- Proxy and port forwarding, makes a remote connection on local port and prints response:
```

Comment:
This is incorrect, nothing is printed from `nc -l local_port | nc hostname remote_port`.

I would reword it like this:

```suggestion
- Proxy and port forwarding, forward data from a local TCP port to the given remote host:
```

Reply:
Updated.